incomplete-cat-B-items.md

```diff
@@ -0,0 +1,116 @@
+|  B  |          | 2.11.3    | menu                            | `submenuShowDelay`                                      |
+|  B  |          | 2.11.6    | menu                            | `file`                                                  | TODO: is this supported already?
+|  B  |          | 3.3.14    | mouse context                   | `Icon`                                                  |
+|  B  |          | 3.4.1     | mouse button                    | `Middle`                                                |
+|  B  |          | 3.4.1     | mouse button                    | `Up` (scroll)                                           |
+|  B  |          | 3.4.1     | mouse button                    | `Down` (scroll)                                         |
+|  B  |          | 3.5.4     | mouse event                     | `Release`                                               |
+|  B  |          | 4.5       | global action                   | `DirectionalCycleWindows`                               |
+|  B  |          | 4.6       | global action                   | `DirectionalTargetWindows`                              |
+|  B  |          | 4.8       | global action                   | `AddDesktop`                                            |
+|  B  |          | 4.9       | global action                   | `RemoveDesktop`                                         |
+|  B  |          | 4.10      | global action                   | `ToggleShowDesktop`                                     |
+|  B  |          | 4.13      | global action                   | `Restart`                                               |
+|  B  |          | 4.15      | global action                   | `SessionLogout`                                         |
+|  B  |          | 5.3       | window action                   | `Lower`                                                 |
+|  B  |          | 5.4       | window action                   | `RaiseLower`                                            |
+|  B  |          | 5.5       | window action                   | `UnFocus`                                               |
+|  B  |          | 5.6       | window action                   | `FocusToBottom`                                         |
+|  B  |          | 5.9       | window action                   | `ToggleShade`                                           |
+|  B  |          | 5.10      | window action                   | `Shade`                                                 |
+|  B  |          | 5.11      | window action                   | `Unshade`                                               |
+|  B  |          | 5.14      | window action                   | `Maximize`                                              |
+|  B  |          | 5.15      | window action                   | `Unmaximize`                                            |
```

Comment:
As we already have `ToggleMaximize`, do we want to implement `(un)Maximize` at all?